resources-plugin/src/main/kotlin/ResourcesPlugin.kt

```diff
@@ -87,6 +88,13 @@ class ResourcesPlugin : KotlinCompilerPluginSupportPlugin {
             )
         }
 
+        // Do not copy resource duplicates, due to we rely on our copyResources task
+        if (isJsBrowserCompilation(kotlinCompilation)) {
+            project.tasks.named("jsTestProcessResources", ProcessResources::class.java) { task ->
+                task.duplicatesStrategy = DuplicatesStrategy.EXCLUDE
+            }
+        }
```

Comment:
I'm not a fan of using `jsTestProcessResources` directly here. I've avoided hardcoded references to tasks across this project, constructing them from references where needed.

I don't understand why this is needed, though. `setupProxyResourcesTask` already sets the same strategy.